vortex-array/src/compute/list.rs

```diff
@@ -52,6 +54,15 @@ pub fn list_contains(array: &dyn Array, value: Scalar) -> VortexResult<ArrayRef>
     if &**elem_dtype != value.dtype() {
```

Comment:
Might be worth promoting this to an actual compute function? That way it would have return type assertions